src/material-examples/stepper-types-editable/stepper-type-editable-example.ts

```diff
@@ -0,0 +1,27 @@
+import {Component} from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+
+/**
+ * @title Stepper type - editable
+ */
+@Component({
+  selector: 'stepper-overview-example',
+  templateUrl: 'stepper-overview-example.html',
+  styleUrls: ['stepper-overview-example.css']
+})
+export class StepperTypeEditableExample {
```

Comment:
Make the directory, filenames, and classes `stepper-editable/`, `stepper-editable-example.*` and `StepperEditableExample`